### PR TITLE
Add info about the type of metric and expiry date on Metric Detail Page

### PR DESCRIPTION
**Pull request by meghajain-1711**
Fixes issue #100

Added type of Metric and Expiry Information below the Metric name and description.
The Metric type links to its documentation as well.

Example of ; never expires
image

Example of : expiry date to come
image

Example of : expiry date gone past
image

@wlach your comments are welcome :)

### CODE REVIEW

**Comment by DawoodShahat** [comment]
Hey @meghajain-1711 , I think you can add these changes to your previous PR [https://github.com/mozilla/glean-dictionary/pull/79](https://github.com/mozilla/glean-dictionary/pull/79) 

Because they are the same pages. 

You would want something like this that has the information like metric_type and metric_expiration date
Screenshot 2020-10-13 at 2 04 02 AM

You can put all the data in one place for a specific metric.

File: src/pages/MetricDetail.svelte [hunk 1]
```diff
@@ -3,9 +3,54 @@
...
+      timing_distribution: `${sourceDocs}timing_distribution.html`,
+      boolean: `${sourceDocs}boolean.html`,
+      labeled_counter: `${sourceDocs}labeled_counters.html`,
+      uuid: `${sourceDocs}uuid.html`,
+      counter: `${sourceDocs}counter.html`,
+      event: `${sourceDocs}event.html`,
+      jwe: `${sourceDocs}jwe.html`,
+    };
+
+    return links[type] || sourceDocs;
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:27]: This seems like a good approach to me, but there's no need to define the sourceDocs variable each time-- instead, try something like this:

```suggestion
    return `${sourceDocs}${links[type]}` || sourceDocs;
```

**Review by wlach — CHANGES_REQUESTED** [review]
> Hey @meghajain-1711 , I think you can add these changes to your previous PR #79
> ...

Yeah, strictly I would agree -- this would be in scope for #79. However, I'm also ok with landing this separately. There will definitely be many followups to all the pages at this early stage, as we figure out what to display and how to display it.

I'm ok to land this once the one issue I mention below is addressed.

**Comment by meghajain-1711** [comment]
Hi @wlach Updated as per the request. It definitely made the codebase simpler :) 
Also , I have added a link to the params.app ie Application from which the metric was found. 

Let me know if it looks good now. 
image

**Review by wlach — APPROVED** [review]

**Pull request merged**